packages/neuron-wallet/src/models/subjects/wallet-deleted-subject.ts

```diff
@@ -1,15 +1,9 @@
 import { Subject } from 'rxjs'
-import ProcessUtils from 'utils/process'
-import { remote } from 'electron'
 
 export default class WalletDeletedSubject {
   private static subject = new Subject<string>()
 
   public static getSubject() {
-    if (ProcessUtils.isRenderer()) {
-      return remote.require('./models/subjects/wallet-delete-subject').default.getSubject()
-    } else {
-      return this.subject
-    }
+    return this.subject
```

Comment:
> We should always use `WalletDeletedSubject` instead of `this` since `this` will be lost in some cases, e.g. destructing

That's a really good find.

Reply:
> IMO, we can introduce a container to manage singletons in the future

Also make sense to me.